README.md

```diff
@@ -13,66 +13,113 @@ Assumptions:
 * You're familiar with opening a terminal on your machine and using the command line as needed.
 
-### Install Prerequisites and Code
+### Installation
+_Get the code and install the runtime and dependencies_
 
 1. If you're not already running Node.js, download and run the installer for your operating system. We build using **Node.js 10.15.3 (LTS)** which we recommend downloading using [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm).
+2. You also need the Node Package Manager (`npm`) CLI. We use version `6.14.5`.
 
-2. You also need npm. We use version `6.14.5`.
-
-```shell
-        npm i -g npm@6.14.5
-```
-
+    ```shell
+    npm i -g npm@6.14.5
+    ```
 3. From the command line, clone the USAspending website repository from GitHub to your local machine:
 
-```shell
-        git clone https://github.com/fedspendingtransparency/usaspending-website.git
-```
-
+    ```shell
+    git clone https://github.com/fedspendingtransparency/usaspending-website.git
+    ```
 4. Change to the `usaspending-website` directory that was created when you cloned the USAspending Website repository.
 
-```shell
-        cd usaspending-website
-```
+    ```shell
+    cd usaspending-website
+    ```
 
 5. Perform a clean install to get an exact dependency tree:
 
-```shell
-        npm ci
-```
+    ```shell
+    npm ci
+    ```
 
-### Configuration File
+### Configuration
+_Alter configuration to non-default values by changing environment variable values._
 
-The USAspending website expects a certain number of environment variables to be defined at runtime. The important ones are listed below along with their default values:
+Several `npm` scripts including webpack bundling of static USAspending website artifacts require environment variables to be defined at script-time, so that their values can be used in script logic or injected into static artifacts. The important ones are listed below along with their default values:
 
-| ENV VAR        | VALUE                            |
-|----------------|----------------------------------|
-| USASPENDING_API| https://api.usaspending.gov/api/ |
-| MAPBOX_TOKEN   | ''                               |
-| GA_TRACKING_ID | ''                               |
+| ENV VAR           | DEFAULT VALUE (if not set)       | PURPOSE 
+|-------------------|----------------------------------|----------------------------------------------------|
+| `ENV`             | dev                              | Determine bundling optimizations and feature flags | 
+| `USASPENDING_API` | https://api.usaspending.gov/api/ | API URL to get backend data                        |
+| `MAPBOX_TOKEN`    | ''                               | Product key for use of MapBox features             |
+| `GA_TRACKING_ID`  | ''                               | Google Analytics key for tracking usage in prod    |
```

Comment:
We also track usage in DEV FWIW

Reply:
I might just leave off the env. Didn't want people to think that if they use this website locally, they're being "tracked".